code/lightna/magento-os-frontend/template/page/head/meta/description.phtml

```diff
@@ -0,0 +1,8 @@
+<?php
+/** @var Lightna\Engine\Data\EntityA $entity */
+
+if (!$entity->metaDescription) {
+    return;
+}
+?>
+<meta name="description" content="<?= $entity->metaDescription()  ?>">
```

Comment:
does it work for cms, product, category?

Reply:
yes